doc/contributing.rst

```diff
@@ -166,20 +166,13 @@ pull requests.
 Building the documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The documentation can be built using sphinx. For that, please additionally
-install the following::
-
-    $ pip install matplotlib sphinx numpydoc sphinx-gallery sphinx_bootstrap_theme sphinx-copybutton pillow joblib psutil nbsphinx
+The documentation can be built using sphinx.
 
 You can build the documentation locally using the command::
 
 $ cd doc/
 $ make html
 
-While MNE is not needed to install hnn-core, as a developer you will need to
```

Comment:
Assuming the pip install works well enough to not need this anymore...

Reply:
I ran through the doc build and it seems like mne needed an extra dependency 'nibabel'. When you install mne with pip you are just installing the minimum required dependencies, there are extra dependencies depending on what you plan to use. I've added 'nibabel' to the doc dependencies in my latest push. I was able to build the docs successfully with just that addition so I don't think we need to add any more.